### PR TITLE
[Tensor] add 1d vocab loss

### PR DESCRIPTION
**Pull request by Wesley-Jzy**
1. reuse col_nn 1d vocab loss in ColoTensor
2. add test in simple_net

### CODE REVIEW

File: colossalai/tensor/_ops/loss.py [hunk 1]
```diff
@@ -12,18 +13,29 @@ def colo_cross_entropy(types, args=(), kwargs=None, pg=None):
...
-    if isinstance(input_tensor, ColoTensor):
-        input_tensor = input_tensor.torch_tensor()
+    if not isinstance(input_tensor, ColoTensor):
+        input_tensor = ColoTensor.init_from_torch_tensor(input_tensor)
     if isinstance(target, ColoTensor):
         target = target.torch_tensor()
 
-    return ColoTensor.init_from_torch_tensor(torch.nn.functional.cross_entropy(input_tensor, target, weight))
+    if input_tensor.is_gathered(): # Input is gathered
+        # TODO(jzy) Shall we make the result of loss function a ColoTensor?
```
Comment (feifeibear) [inline colossalai/tensor/_ops/loss.py:31]: Yes, make it a ColoTensor.

**Review by feifeibear — APPROVED** [review]
